server.js

```diff
@@ -70,6 +70,17 @@ function cropImage(paramSource, res, next){
   });
 };
 
+function computeURL(static_host, u){
```

Comment:
This is mostly there, i think having the ability to not just be a blanket s3.amazonaws.com would be good. I tried this out locally and it worked.

``` javascript
function escapeRegExp(string){
  return string.replace(/([.*+?^=!:${}()|\[\]\/\\])/g, "\\$1");
}

function validUrl(arr, url) {
  return arr.some(function(arrVal) {
    var host_regex = new RegExp(escapeRegExp(arrVal), 'g');
    return url.match(host_regex);
  });
}

function computeURL(static_host, u){
  var whitelists = ['https://s3.amazonaws.com/astronomy-rewind'];
  if (validUrl(whitelists, u)) {
    return u
  } else {
    return static_host + u;
  }
};
```